data/nfv/ws_main.py

```diff
@@ -0,0 +1,1643 @@
+#!/usr/bin/env python
+# encoding: utf-8
+#
+# Hanoh Haim
+# Cisco Systems, Inc.
```

Comment:
This file has copyright for Cisco or it's just stated that guy who wrote it from Cisco ?